bin/gen-crons.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+import os
+from optparse import OptionParser
+
+from jinja2 import Template
+
+
+HEADER = '!!AUTO-GENERATED!! Edit {template}.tmpl instead.'
+TEMPLATE_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', 'etc', 'cron.d'))
+
+
+def main():
+    parser = OptionParser()
+    parser.add_option('-w', '--webapp',
+                      help='Location of web app (required)')
+    parser.add_option('-s', '--source',
+                      help='Location of source for the web app (required)')
+    parser.add_option('-t', '--template',
+                      help='Name of the template (e.g. bedrock-prod)')
+    parser.add_option('-u', '--user', default='root',
+                      help=('Prefix cron with this user. '
+                            'Only define for cron.d style crontabs.'))
+    parser.add_option('-p', '--python', default='python2.6',
+                      help='Python interpreter to use.')
+
+    (opts, args) = parser.parse_args()
+
+    if not opts.webapp:
+        parser.error('-w must be defined')
+
+    if not opts.template:
+        parser.error('-t must be defined')
+
+    ctx = {'django': 'cd %s && %s manage.py' % (opts.webapp, opts.python)}
```

Comment:
nit: seems like this would be better named as 'manage' or 'django_manage' that would make it clearer to read the cron templates.